cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -84,23 +84,28 @@
     {% endif %}
 
     <div class="o-form_group">
-        <label class="a-label a-label__heading" for="comment">
-            {% if value.intro_text %}
-                {{ value.intro_text }}
-            {% elif value.question_text %}
-                {{ value.question_text }}
-            {% elif value.was_it_helpful_text %}
-                {% if page.language == 'es' %}Comentario adicional{% else %}Additional comment{% endif %} <small class="a-label_helper">({% if page.language == 'es' %}opcional{% else %}optional{% endif %})</small>
-            {% endif %}
-        </label>
-        <p class="u-mb15"><small><em>{% if page.language == 'es' %}Nota: No incluya información confidencial, como su nombre, información de contacto, número de cuenta o número de seguro social en este campo.{% else %}Note: Do not include sensitive information like your name, contact information, account number, or social security number in this field.{% endif %}
-        </em></small></p>
-        <textarea class="a-text-input" id="comment" name="comment" rows="6"
-                 {% if not value.was_it_helpful_text %}
-                     required
-                 {% endif %}>
-            {{- form.comment.value() | default('', true) -}}
-        </textarea>
+        <div class="m-form-field">
+            <label class="a-label a-label__heading" for="comment">
+                {% if value.intro_text %}
+                    {{ value.intro_text }}
+                {% elif value.question_text %}
+                    {{ value.question_text }}
+                {% elif value.was_it_helpful_text %}
+                    {% if page.language == 'es' %}Comentario adicional{% else %}Additional comment{% endif %} <small class="a-label_helper">({% if page.language == 'es' %}opcional{% else %}optional{% endif %})</small>
+                {% endif %}
+            </label>
+            <p class="u-mb15"><small><em>{% if page.language == 'es' %}Nota: No incluya información confidencial, como su nombre, información de contacto, número de cuenta o número de seguro social en este campo.{% else %}Note: Do not include sensitive information like your name, contact information, account number, or social security number in this field.{% endif %}
+            </em></small></p>
+            <textarea class="a-text-input a-text-input__full"
+                      id="comment"
+                      name="comment"
+                      rows="6"
+                      {% if not value.was_it_helpful_text %}
```

Comment:
Trivial: Make this `{%-` to bring the `>` up next to the `rows` attribute if the condition evaluates to false.